samples/Combobox.html

```diff
@@ -42,9 +41,12 @@
 			ignoreCase.on("change", function () {
 				comboTeams.ignoreCase = ignoreCase.checked;
 			});
-
-			$("input[type=number]").on("input", function(evt) {
+			var onInput = function (evt) {
 				comboTeams.minFilterChars = parseInt(evt.target.value) || 0;
+			}
```

Comment:
You could actually declare this as

```js
function onInput(evt) {
```
and save a few characters.

As you have it here you are missing a semicolon.